src/earn/saga.test.ts

```diff
@@ -416,18 +419,21 @@ describe('depositSubmitSaga', () => {
 })
 
 describe('withdrawSubmitSaga', () => {
-  const mockRewards = [{ tokenId: mockArbArbTokenId, amount: '1' }]
+  const mockRewards = [
+    { tokenId: 'arbitrum-sepolia:0x912ce59144191c1204e64559fe8253a0e49e6548', amount: '0.01' },
+  ]
+  const mockPool = mockRewardsPositions[0] as EarnPosition
```

Comment:
For my understanding, `mockRewardsPositions` is a mock of what a pool position with a reward would return, so we are able to use the first item as the normal pool position (and then the second is the reward position object)?

Reply:
Correct ✅